plugins/outputs/warp10/warp10.go

```diff
@@ -0,0 +1,198 @@
+package warp10
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/influxdata/influxdb/client/v2"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+type Warp10 struct {
+	Prefix string
+
+	WarpUrl string
+
+	Token string
+
+	Debug bool
+}
+
+var sampleConfig = `
+  # prefix for metrics class Name
+  prefix = "telegraf."
+  ## Telnet Mode ##
```

Comment:
what does "telnet mode" mean? if this is an http request it doesn't sound like it would be telnet?